src/FakeItEasy/Configuration/RuleBuilder.cs

```diff
@@ -63,7 +63,6 @@ public IVoidConfiguration WhenArgumentsMatch(Func<ArgumentCollection, bool> argu
 
         public virtual IAfterCallSpecifiedConfiguration DoesNothing()
         {
-            this.RuleBeingBuilt.Applicator = x => { };

```

Comment:
Do we we have any specs which verify that the sequence of actions described above work in the way described?